js/Particles.js

```diff
@@ -8,7 +8,8 @@ PHOTONS.RangeType = Object.freeze( {
 
 	Cube: 1,
 	Sphere: 2,
-	Plane: 3
+	Plane: 3,
+	Integer: 4

```

Comment:
I agree with you.  I had thought to specify that as "Line" before, only a line might still imply a 3 value vector rather than a single numerical value.  What do you recommend?